packages/hdwallet-provider/src/index.ts

```diff
@@ -177,7 +177,7 @@ class HDWalletProvider {
     payload: JSONRPCRequestPayload,
     callback: JSONRPCErrorCallback | Callback<JsonRpcResponse>
   ): void {
-    return this.engine.send.call(this.engine, payload, callback);
+    return this.engine.sendAsync.call(this.engine, payload, callback);
```

Comment:
really?

Reply:
Ya, have an alternative in mind?

Reply:
What do you mean by really? @CruzMolina can you explain exactly what this is about?

Reply:
Need to look into this a bit more. Web3 might not have actually dropped `sendAsync`.

The problem is Web3 `1.x` came out and formally removed support for `sendAsync`. `web3-provider-engine` responded by removing support for sync requests & forcing providers to have a `sendAsync` method.

Reply:
Idk, the options here kind've suck.

We could downgrade Web3 here to `0.x` & not have to do the hack on this line (or the shimming).